omegaconf/listconfig.py

```diff
@@ -379,8 +384,24 @@ def _get_node(
             assert isinstance(self.__dict__["_content"], list)
             if validate_access:
                 self._validate_get(key)
-            return self.__dict__["_content"][key]  # type: ignore
+            assert self.__dict__["_content"] is not None
+            assert not isinstance(self.__dict__["_content"], str)
```

Comment:
I am confused by these three assertions:
  - `isinstance(_content, list)`
  - `_content is not None`
  - `not isinstance(_content, str)`

Could these maybe be replaced with a single assertion?

Reply:
On my computer, the tests + mypy still pass if the later two assertions are deleted.

Reply:
Good spot, the first one is sufficient.